app/src/main/java/sdp/moneyrun/menu/JoinGameImplementation.java

```diff
@@ -268,10 +277,24 @@ public void onCancelled(@NonNull DatabaseError error) {
     }
 
     private void joinLobbyFromJoinButton(GameRepresentation gameRepresentation){
+        DatabaseReference gamePlayers = databaseReference.child(activity.getString(R.string.database_games)).child(gameRepresentation.getGameId()).child(activity.getString(R.string.database_open_games_players));
```

Comment:
Could you also try to test this function if possible?

Reply:
Right, that is part of why I tried to be thorough with all the other testing in order to reach 80%. The issue is that the joinGameButton is enabled only when you are geographically close to it. Locally, I can test it since I can set the location of the emulator so that it matches the one of a game and you can click it. Now, after long research I still have not found a way to set the location of the emulator when cirrus runs it. 

I guess a good intermediary would be to write an instrumentation test that you can execute locally (after having set the location of your emulator), but comment it out so that cirrus doesn't execute it. With that, we can have the confidence that it works.